app/assets/javascripts/discourse/app/lib/plugin-api.js

```diff
@@ -2014,6 +2018,31 @@ class PluginApi {
   registerHashtagSearchParam(param, context, priority) {
     registerHashtagSearchParam(param, context, priority);
   }
+
+  /**
+   * Add a new button to be overlaid ontop of a table appearing in a post.
+   * On wider tables the button will be appending along with the Expand table button.
+   *
+   * Example usage:
+   *
+   *  api.addTableWrapperButton(
+   *    themePrefix("discourse_table_builder.edit.btn_edit"),
+   *    "pencil-alt",
+   *    ["btn-edit-table"],
+   *    generateModalEvent
+   * );
+   *
+   * @param {string} label - An i18n key for the button label
+   * @param {icon} icon - An icon name from fontawesome
```

Comment:
Should be string no? We don't have "icon" type

```suggestion
   * @param {string} icon - An icon name from fontawesome
```

Reply:
A yes good catch, thanks!